snowtrick/public/js/detail.js

```diff
@@ -0,0 +1,83 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+        // add event on trigger button
+    function activeTrashButton() {
+        const trash_btn = document.querySelectorAll(".trash_btn");
+        trash_btn.forEach((trash) => {
+            trash.addEventListener('click', (event) => {
+                let myInit = {
+                    method: trash.data.method,
+                    headers: {'content-type': 'application/json'}
+                };
+
+                fetch(trash.dataset.path, myInit)
+                    .then((res) => {
+                        document.querySelector("#" + trash.dataset.entity + "_" + trash.dataset.id).outerHTML = "";
+                        closeAllModals();
+                    });
+            })
```

Comment:
Codacy has a fix for the issue: [Replace `············})` with `······});`](https://app.codacy.com/gh/trousse/OC_Projet6/pullRequest?prid=10057695)
```suggestion
      });
```